### PR TITLE
Typofixes

### PR DESCRIPTION
**Pull request by nexxai**

### CODE REVIEW

File: admin/index.php [hunk 1]
```diff
@@ -227,7 +227,7 @@
 		$display_on_page = ( $offset + 1 );
 	}
 
-	// Determing Total Amount Of Pages
+	// Determining Total Amount Of Pages
```
Comment (dgw) [inline admin/index.php:230]: I see where you came from with the existing change, but in context (and based on the other comments above) this is the intended wording:

```suggestion
	// Determine Total Amount Of Pages
```

Comment (nexxai) [reply admin/index.php]: Agreed.  Updated to use your suggestion.

**Review by dgw — APPROVED** [review]
LGTM

**Comment by nexxai** [comment]
My apologies, I deleted the repo by accident but the changes should still be merged.

**Review by LeoColomb — APPROVED** [review]

**Pull request merged**